src/service/resources-impl.js

```diff
@@ -807,7 +807,7 @@ export class Resources {
               const box = request.resource.getLayoutBox();
               minTop = minTop == -1 ? box.top : Math.min(minTop, box.top);
               request.resource./*OK*/changeSize(
-                  request.newHeight, request.newWidth);
+                  request.newHeight, request.newWidth, request.callback);
```

Comment:
Is there a special reason that we don't pass in `request.callback` after scroll here? @dvoytenko @jridgewell